examples/ModelComparison/ModelComparison.ipynb

```diff
@@ -0,0 +1,345 @@
+      "    prompt: \u001b[32m'(1) - true or false'\u001b[39m\n",
+      "  },\n",
+      "  {\n",
+      "    model: \u001b[32m'gpt-3.5-turbo'\u001b[39m,\n",
+      "    temperature: \u001b[33m0\u001b[39m,\n",
+      "    prompt: \u001b[32m'(2) - Answer using true or false only'\u001b[39m\n",
+      "  },\n",
+      "  {\n",
+      "    model: \u001b[32m'gpt-3.5-turbo'\u001b[39m,\n",
+      "    temperature: \u001b[33m0\u001b[39m,\n",
+      "    prompt: \u001b[32m\"(3) - Answer the following question as accurately as possble with the words 'true' or 'false' in lowercase only. Do not include any other words in the response\"\u001b[39m\n",
+      "  },\n",
+      "  {\n",
+      "    model: \u001b[32m'gpt-3.5-turbo'\u001b[39m,\n",
+      "    temperature: \u001b[33m0.1\u001b[39m,\n",
+      "    prompt: \u001b[32m'(1) - true or false'\u001b[39m\n",
+      "  },\n",
+      "  {\n",
+      "    model: \u001b[32m'gpt-3.5-turbo'\u001b[39m,\n",
+      "    temperature: \u001b[33m0.1\u001b[39m,\n",
+      "    prompt: \u001b[32m'(2) - Answer using true or false only'\u001b[39m\n",
+      "  }\n",
+      "]\n"
+     ]
+    }
+   ],
+   "source": [
+    "const combinations: { model: string; temperature: number; prompt: string }[] =\n",
+    "  [];\n",
+    "for (const model of [\"gpt-3.5-turbo\", \"gpt-4\", \"gpt-4o\"]) {\n",
+    "  for (const temperature of [\n",
+    "    0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1,\n",
+    "  ]) {\n",
+    "    for (const prompt of prompts) {\n",
+    "      combinations.push({\n",
+    "        model,\n",
+    "        temperature,\n",
+    "        prompt,\n",
+    "      });\n",
+    "    }\n",
+    "  }\n",
+    "}\n",
+    "console.log(combinations.slice(0, 5));\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's use the functions and data that we have set up to run some evals on Braintrust! We will be using two scorers for this eval:\n",
+    "\n",
+    "1. A simple exact match scorer that will compare the output from the LLM exactly with the expected value\n",
+    "2. A Levenshtein scorer which will calculate the Levenshtein distance\n",
+    "\n",
+    "We are also adding the model, temperature, and prompt into the metadata so that we can use those fields to help our visualization inside the braintrust app after the evals are finished running.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "import { Eval } from \"braintrust\";\n",
+    "import { Levenshtein } from \"autoevals\";\n",
+    "\n",
+    "const exactMatch = (args: { input; output; expected? }) => {\n",
+    "  return {\n",
+    "    name: \"ExactMatch\",\n",
+    "    score: args.output === args.expected ? 1 : 0,\n",
+    "  };\n",
+    "};\n",
+    "\n",
+    "// feel free to loop differently instead depending on your rate limits\n",
```

Comment:
Would this potentially report experiments out of order?

I think for the purposes of the demo the graph looks nicer when things run in order since it default sorts by time, but I'm happy to switch it over if I've misunderstood the way it works or if you don't think the ordering matters

Reply:
Yep it could report things out of order. Understand the desire to make them have a nice/predictable order though.

How long does it take to run on your machine?

Reply:
Looks like gpt calls take ~5 seconds to run over the 5 examples. So with 60 combinations ((2 gpt + 2 claude) * 3 prompts * 5 temperatures), it would take about 5 minutes.

Reply:
I have a mild preference for parallelizing it, just because I think people really appreciate the speed/parallelization aspect of Braintrust, but I don't feel super strongly about it. If you keep it serialized, then I would add a comment here explaining why you're doing that, so that someone who cares less won't have a slow experience.